src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -352,7 +352,7 @@ def test_tag_update_by_patch(self, resource_group, resource_group_location):
 
         # Test Microsoft.Resources/resourceGroups
         self.cmd('resource tag --ids {resource_group_id} --tags {tag}',
-                 checks=self.check('tags', {'StorageType': 'Standard_LRS', 'cli-test': 'test', 'type': 'test'}))
+                 checks=self.check('tags', {'StorageType': 'Standard_LRS', 'cli-test': 'test', 'type': 'test'})) 
```

Comment:
nit: Remove trailing whitespace.

You might want to run `azdev linter resource` and `azdev style resource` and make sure all checks pass locally.